docs/chain/linking.mdx

```diff
@@ -0,0 +1,17 @@
+---
+id: linking
+title: Link DID
+---
+
+<head>
+  <script>window ||= new Object(); window.kilt = new Object()</script>
+</head>
+
+You may link your on-chain DID, with your new web3name, to any accounts you have in the Polkadot ecosystem.
+
+In this way, people with your Polkadot ecosystem account address will be able to see your DID and your web3 name and, vice versa, those with your web3name can see your public address. This is especially useful for validators and collators, who can provide a personalized name that makes them easily identifiable.
```

Comment:
To make diffs and reviews easier readable we always have a single sentence per line. Markdown renders a single new line as a space. So this change wouldn't change the layout.

```suggestion
In this way, people with your Polkadot ecosystem account address will be able to see your DID and your web3 name and, vice versa, those with your web3name can see your public address.
This is especially useful for validators and collators, who can provide a personalized name that makes them easily identifiable.
```